apps/cookbook/src/app/iphone/iphone.component.scss

```diff
@@ -1,5 +1,6 @@
 @use '~@kirbydesign/core/src/scss/utils';
 
+/* stylelint-disable custom-property-pattern */
```

Comment:
Why do we disable the `custom-property-pattern` rule?

Reply:
This is because the custom properties are named something like `--text-color--dark`, with a double dash between color and dark. 

I don't think they are used anywhere though, could you double-check? Then we could just remove them completely, and get rid of the rule.

Reply:
Ah, ok. `--text-color--dark` is being used in [one place in this file](https://github.com/kirbydesign/designsystem/blob/0e15ad9161a9e3d3c02222e25813f7545fad5b8a/apps/cookbook/src/app/iphone/iphone.component.scss#L78), so we should probably still disable the rule.

But we should also revisit the naming of our custom properties so we don't have to disable the rule here.